docs/tests/integration/test_wazuh_db/index.md

```diff
@@ -1,3 +1,13 @@
 # Overview
 
+**Wazuh-db** is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket so receive requests and provide information.
 
+## Tiers
+### Tier 0
+#### Test wazuh_db
+
+These tests check the Wazuh-db commands syntax, valid responses and error messages in different circumstances. They are divided in **.yaml** configuration files:
+
+- **agent_messages.yaml**: tests for agents in general, currently only for Agents' CVEs table
```

Comment:
I think we can avoid this part: _currently only for Agents' CVEs table_

i.e.:
_tests for agents DB, general commands._

Reply:
Fixed!